Makefile

```diff
@@ -32,7 +32,8 @@ authorize:
 	@@source .env; ./lib/oc_add_gh_team_to_nsp.sh --token $$GH_TOKEN -t $$GH_DEVELOPERS_TEAM -pp $$OC_PROJECT_PREFIXES -r view
 	# Create cas-namespaces secret in all dev-test-prod namespaces
 	@@source .env; ./lib/oc_create_namespace_secret.sh -pp $$OC_PROJECT_PREFIXES -ap $$AIRFLOW_PREFIX -gp $$GGIRCS_PREFIX -cp $$CIIP_PREFIX
-
+	# Create dockerhub-registry secret in all dev-test-prod-tools namespaces
+	@@source .env; ./lib/oc_create_dockerhub_secret.sh -pp $$OC_PROJECT_PREFIXES -ps dev,test,prod,tools -u $$DOCKER_USERNAME -p $$DOCKER_PASSWORD 
```

Comment:
Let's move these two scripts to a different make target (`make provision`?). The rolebinding synchronization with GH teams should only be run by the technical lead, whose access to the namespace is controlled via annotations. If another team member runs it, they will lose access to the namespaces, since the first part of those scripts is to delete all rolebindings that were previously created (to ensure synchronization).